_source/_posts/2021-07-14-flask-scim-server.md

```diff
@@ -0,0 +1,528 @@
+---
+layout: blog_post
+title: "Okta, SCIM and Flask"
+author: cale-switzer
+by: advocate
+communities: [python]
+description: "Using a Flask SCIM server, learn more about the how & why behind how Okta implements the provisioning protocol."
+tags: [scim, python, flask]
+tweets:
+- ""
+- ""
+- ""
+image:
+type: awareness
+---
+
+At a very high level, the SCIM (System for Cross-Identity Management) protocol exists to provision and sync users and groups across various, independent systems. There is some room for nuance in the [SCIM spec](https://datatracker.ietf.org/doc/html/rfc7644). In this post we will take a look at some basic SCIM operations and what the flow looks like under the hood between Okta and a SCIM server built in Flask using PostgreSQL as our app database. Even if you're not a Python dev, hopefully you can take away some of the concepts in this guide for your own SCIM implementation.
+
+### What you'll need
```

Comment:
```suggestion
### What You'll Need
```
we'e using title case in top-level headers

Reply:
yeesh - my bad! Brian corrected me on one of my titles above and I should've applied the changes to all of my titles.